IO.Eventuate.Tram/Consumer/Kafka/SwimlaneBasedDispatcher.cs

```diff
@@ -1,13 +1,17 @@
 using System;
 using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
 using IO.Eventuate.Tram.Messaging.Common;
 using Microsoft.Extensions.Logging;
 
 namespace IO.Eventuate.Tram.Consumer.Kafka
 {
 	public class SwimlaneBasedDispatcher
 	{
-		private readonly ConcurrentDictionary<int, SwimlaneDispatcher> _map = new ConcurrentDictionary<int, SwimlaneDispatcher>();
+		private bool _dispatcherStopped;
+
+		private readonly ConcurrentDictionary<int, SwimlaneDispatcher> _map = new();
```

Comment:
Consider upgrading the name to give a hint as to its purpose

Reply:
Renamed to __swimlaneDispatchersBySwimlane